geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/QueryStrategyDecider.scala

```diff
@@ -21,13 +20,12 @@ import org.opengis.filter.{And, Filter, Id, PropertyIsLike}
 import scala.collection.JavaConversions._
 
 trait VersionedQueryStrategyDecider {
-  def chooseStrategy(sft: SimpleFeatureType, query: Query, hints: StrategyHints, version: Int): Strategy
+  def chooseStrategy(sft: SimpleFeatureType, query: Query, hints: StrategyHints): Strategy
```

Comment:
The version is kind of obsolete since we now have a different impl of the strategy decider depending on the data version. We can always add it back in later if needed... this is an internal API.